package-manifest.toml

```diff
@@ -271,8 +271,8 @@ only_for_targets.image_type = "standard"
 # 3. Use source.type = "manual" instead of "prebuilt"
 source.type = "prebuilt"
 source.repo = "dendrite"
-source.commit = "67a45ff847989e8b8be37c099f70a7451b277c0d"
-source.sha256 = "633e71a764fe02adf2c62dff31923ca4f8863818d0e4d68983c8a360bf23eb3a"
+source.commit = "8ba0f2cde2577219f58eb768850705e9ef7249a0"
+source.sha256 = "52772b64689a590663716bcd79d10e9013f31a05f2315472495c09241069eb47"
```

Comment:
https://buildomat.eng.oxide.computer/public/file/oxidecomputer/dendrite/image/8ba0f2cde2577219f58eb768850705e9ef7249a0/dendrite-asic.sha256.txt says this should be:

b1efb112820a682a4fd651825a835d35c5a55cc15fc7898266bbd77c2f48c18a